instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -0,0 +1,263 @@
+---
+rules:
+  - bean: org.apache.camel:context=*,type=context,name=*
+    prefix: camel.context.
+    metricAttribute:
+      context: param(context)
+      camelVersion: beanattr(CamelVersion)
+    mapping:
+      ExchangesCompleted:
+        metric: exchange.completed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges processed successfully since context start-up or the last reset operation.
+      ExchangesFailed:
+        metric: exchange.failed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges that failed to process since context start-up or the last reset operation.
+      ExchangesInflight:
+        metric: exchange.inflight
+        type: updowncounter
+        unit: "{exchanges}"
+        desc: Indicates the number of exchanges currently transiting the context.
+      ExchangesTotal:
+        metric: exchange
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges, passed or failed, processed context start-up or the last reset operation.
```

Comment:
see https://github.com/open-telemetry/semantic-conventions/issues/50

but since there's no decision on that issue, I think this is ok

Reply:
I prefer to change it in a way to comply with the convention whatever decision will be made.